server/static/static.go

```diff
@@ -52,7 +54,16 @@ func NewStaticFileServer(env environment.Env, fs fs.FS, rootPaths []string) (*St
 			return nil, err
 		}
 
-		handler = handleRootPaths(env, rootPaths, template, version.AppVersion(), handler)
+		jsPath := *jsEntryPointPath
+		if appBundleHash != "" {
+			jsPath = strings.ReplaceAll(jsPath, "{APP_BUNDLE_HASH}", appBundleHash)
+		}
+
+		if strings.HasPrefix(jsPath, "http://") || strings.HasPrefix(jsPath, "https://") {
+			env.GetHealthChecker().AddHealthCheck("static_bundle", &healthChecker{jsPath: jsPath})
```

Comment:
uber nit: rename the health check to `"app_static_file_server"` (since the server handles stuff other than the static bundle)

Reply:
Done